sugoi-api-test/src/test/resources/scenario/user_scenario_sync.feature

```diff
@@ -230,3 +230,9 @@ Feature: User scenario
         When the client perform POST request on url /realms/domaine1/users/nogroup/reinit-password
         Then the client receives status code 400
 
+    Scenario: Get user with group filter
+        When the client perform GET request on url /realms/domaine1/storages/default/users?groups=Managers_Applitest
+        And show body received
+        Then the client receives status code 200
+        Then the client expect to receive a list of users
```

Comment:
Can you add more checks in this test ? like a user in managers is present, a user not in manager is absent